sdk/typescript/src/types/transactions.ts

```diff
@@ -47,7 +47,7 @@ export type GasCostSummary = {
 
 export type ExecutionStatus =
   | { Success: { gas_cost: GasCostSummary } }
-  | { Failure: { gas_cost: GasCostSummary; error: string } };
+  | { Failure: { gas_cost: GasCostSummary; error: any } };
```

Comment:
@Jibz1 You will see a console error from the sdk in the main branch in the landing page. This PR will fix that.